### ISSUE TITLE
how to use?

### ISSUE THREAD

**Issue by ohyeah521** [MIuserX/secure_socks#1]
please tell me how to use it,thank you.

**Comment by MIuserX** [comment]
Oh, my God.
It's surprise for me that someone likes my works.

Unfortunately, I have given up this project.
Because I found a better way to do what I want.

My situation goes like this:
1) a socks5 proxy (it doesn't belong to me but I can use)
2) a vps (it belongs to me)

I want that my traffic goes with the path like this:
web browser <-> a proxy client on localhost(encrypt/decrypt) <-> the socks5 proxy <-> a proxy server on my vps(encrypt/decrypt) <-> the target websites

As we all know, a socks5 proxy transfers unencrypted data, it's unsafe.
So I need a tool which can encrypt/decrypt data on my localhost and my vps.
I don't know if your situation is the same as mine.
I found such tool on github later.
Its link is: https://github.com/ginuerzh/gost

May this can help you.
